src/stylesString.js

```diff
@@ -11,27 +11,22 @@ const complexStylesString = `
   cursor: pointer;
   float: left;
   font-family: inherit;
-      font-size: 14px;
+  font-size: 14px;
   font-weight: 400;
   height: 42px;
   line-height:    40px;
   outline: 0;
   padding-left    : 18px;
   padding-right: 30px;
   ;
-
   ;
```

Comment:
There are redundant semicolons on these lines. They should be removed to avoid confusion and maintain clean code.